tests/unit/url_responses/auth_github_responses_sync_label.yaml

```diff
@@ -0,0 +1,50 @@
+# this file is generated automatically via responses. Use _recorder.record()
+responses:
+- response:
+    auto_calculate_content_length: false
+    body: '{"id":35534068,"account":{"login":"beliaev-maksim","id":51964909,"node_id":"MDQ6VXNlcjUxOTY0OTA5","avatar_url":"https://avatars.githubusercontent.com/u/51964909?v=4","gravatar_id":"","url":"https://api.github.com/users/beliaev-maksim","html_url":"https://github.com/beliaev-maksim","followers_url":"https://api.github.com/users/beliaev-maksim/followers","following_url":"https://api.github.com/users/beliaev-maksim/following{/other_user}","gists_url":"https://api.github.com/users/beliaev-maksim/gists{/gist_id}","starred_url":"https://api.github.com/users/beliaev-maksim/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/beliaev-maksim/subscriptions","organizations_url":"https://api.github.com/users/beliaev-maksim/orgs","repos_url":"https://api.github.com/users/beliaev-maksim/repos","events_url":"https://api.github.com/users/beliaev-maksim/events{/privacy}","received_events_url":"https://api.github.com/users/beliaev-maksim/received_events","type":"User","site_admin":false},"repository_selection":"selected","access_tokens_url":"https://api.github.com/app/installations/35534068/access_tokens","repositories_url":"https://api.github.com/installation/repositories","html_url":"https://github.com/settings/installations/35534068","app_id":308591,"app_slug":"syncronize-issues-to-jira","target_id":51964909,"target_type":"User","permissions":{"issues":"write","contents":"read","metadata":"read"},"events":["issues","issue_comment"],"created_at":"2023-03-22T14:24:41.000Z","updated_at":"2023-03-22T14:25:28.000Z","single_file_name":null,"has_multiple_single_files":false,"single_file_paths":[],"suspended_by":null,"suspended_at":null}'
+    content_type: text/plain
+    method: GET
+    status: 200
+    url: https://api.github.com:443/repos/beliaev-maksim/test-ci/installation
```

Comment:
I do not think we need to duplicate responses for authentication.

I handled this part in another PR, if you rebase your branch on main, you can just follow the example to add your own base64 string

all the rest in the PR looks good. Once this tiny bit is fixed, I can double test on my machine and we can merge. Thanks!